policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/non_refundable.yaml

```diff
@@ -0,0 +1,7 @@
+description: Non-refundable tax credits in Ontario.
+values:
+  2022-01-01:
+    - yt_childrens_arts_amount
```

Comment:
where does it say this is a non-refundable credit?

Reply:
[Source ](https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/yukon/5011-pc/information-residents-yukon.html) - lists the program under "Part B" -> yukon non refundable credits

Reply:
OK could you add https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/yukon/5011-pc/information-residents-yukon.html#P4_partb as a reference? then git pull upstream master will fix the test failure